snmp/osupdate

```diff
@@ -28,39 +28,39 @@ CMD_PACMAN='-Sup'
 ################################################################
 # Don't change anything unless you know what are you doing     #
 ################################################################
-if [ -f "$BIN_ZYPPER" ]; then
+if [ -f '/etc/SuSE-release' ]; then
     # OpenSUSE
     UPDATES=`$BIN_ZYPPER $CMD_ZYPPER | $BIN_WC $CMD_WC`
     if [ $UPDATES -ge 2 ]; then
         echo $(($UPDATES-2));
     else
         echo "0";
     fi
-elif [ -f "$BIN_DNF" ]; then
+elif [ -f '/etc/fedora-release' ]; then
     # Fedora
     UPDATES=`$BIN_DNF $CMD_DNF | $BIN_WC $CMD_WC`
     if [ $UPDATES -ge 1 ]; then
         echo $(($UPDATES-1));
     else
         echo "0";
     fi
-elif [ -f "$BIN_PACMAN" ]; then
+elif [ -f '/etc/arch-release' ]; then
     # Arch
     UPDATES=`$BIN_PACMAN $CMD_PACMAN | $BIN_WC $CMD_WC`
     if [ $UPDATES -ge 1 ]; then
         echo $(($UPDATES-1));
     else
         echo "0";
     fi
-elif [ -f "$BIN_YUM" ]; then
+elif [ -f '/etc/redhat-release' ]; then
```

Comment:
This doesn't cover centos.

Reply:
It sure does. 

`cat /etc/redhat-release
CentOS Linux release 7.5.1804 (Core)`

Reply:
Let me be more generic.  This doesn't include some random distro that uses yum and doesn't have /etc/redhat-release.  Replace yum and /etc/redhat-release with any combo.

Reply:
Made the changes